hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/portable/PortableGenericRecord.java

```diff
@@ -22,6 +22,7 @@
 import com.hazelcast.nio.serialization.GenericRecord;
 import com.hazelcast.nio.serialization.HazelcastSerializationException;
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+import org.jetbrains.annotations.NotNull;
```

Comment:
`javax.annotation.Nonnull`?

Reply:
Interesting. I have added `javax.annotation.Nonnull` to `GenericRecord` and then IDE put the annotations to subclasses automatically. It seems I should not trust the IDE. Correcting right away.